src/PointwiseFunctions/Hydro/Tags.hpp

```diff
@@ -66,6 +66,13 @@ struct LorentzFactor : db::SimpleTag {
   static std::string name() noexcept { return "LorentzFactor"; }
 };
 
+/// The square of the Lorentz factor \f$W^2\f$.
+template <typename DataType>
+struct LorentzFactorSquared {
+  using type = Scalar<DataType>;
+  static std::string name() noexcept { return "LorentzFactorSquare"; }
```

Comment:
The struct name and the text in the string should match (LorentzFactorSquare*d*)